2020/09/09/introduction-to-borg-backup.html.md

```diff
@@ -0,0 +1,96 @@
+---
+author: "Kannan Ponnusamy"
+title: "Introduction to BorgBackup"
+tags: linux, sysadmin, backups
+---
+![black and silver hard disk drive photo](/blog/2020/09/09/introduction-to-borg-backup/image-1.jpg)
+[Photo](https://unsplash.com/photos/ShHkXuZdpTw) by [Frank R](https://unsplash.com/@frank041985)
+
+### What is Borg?
+BorgBackup (also referred to as: Borg) is a ‘deduplicating’ backup program that eliminates duplicate or redundant information. Optionally, it can also support compression and authenticated encryption. 
+
+The main objective of Borg is to provide an efficient and secure way to backup data. The deduplication technique utilized to produce the backup process is very quick and effective. 
+
+#### Step 1 - How to install the Borg backups
+On Ubuntu:
+
+`apt install borgbackup`
```

Comment:
These code blocks (all throughout the article) should all look like ... well, I can't put it here because GitHub uses Markdown too! But look at another blog post such as `2020/08/27/containerizing-magento-with-docker-compose-elasticsearch-mysql-and-magento.html.md` for the blocks that start and end with three backticks, optionally with a syntax-highlighting annotation at the start.

That's also easier than a single backtick at the start and end of each line.

Reply:
Makes sense, working on the changes.